server/controllers/allocationsController.ts

```diff
@@ -1,11 +1,12 @@
 import { Request, Response } from 'express'
+import dayjs from 'dayjs'
 import AllocationsService from '../services/allocationsService'
 
 export default class AllocationsController {
   constructor(private readonly allocationsService: AllocationsService) {}
 
   async getAllocations(req: Request, res: Response): Promise<void> {
     const response = await this.allocationsService.getUnallocatedCases(res.locals.user.token)
-    res.render('pages/index', { unallocatedCases: response.length > 99 ? '99+' : response.length })
+    res.render('pages/index', { unallocatedCases: response, dayjs })
```

Comment:
I liked this in the controller rather than the template - controller logic tends to be easier to test and reuse - but it's not a big deal

Reply:
Yep agree, I'll update 👍